cli.js

```diff
@@ -31,21 +31,21 @@ if (args.version) {
 }
 
 if (args.help || args._.length === 0) {
-  console.log('\nUsage: dependency-check <path to package.json or module folder> <additional entries to add> <options>')
+  console.log('\nUsage: dependency-check <path to module files, package.json or module folder> <additional entries to add> <options>')
 
   console.log('\nOptions:')
   console.log('--missing (default)   Check to make sure that all modules in your code are listed in your package.json')
   console.log('--unused, --extra     The inverse of the --missing check and will tell you which modules in your package.json *were not* used in your code')
   console.log("--no-dev              Won't tell you about devDependencies that are missing or unused")
   console.log("--no-peer             Won't tell you about peerDependencies that are missing or unused")
   console.log("--ignore-module, -i   Won't tell you about these module names when missing or unused. Supports globbing")
-  console.log('--entry               By default your main and bin entries from package.json will be parsed, but you can add more the list of entries by passing them in as --entry')
-  console.log("--no-default-entries  Won't parse your main and bin entries from package.json")
+  console.log('--entry               If a package.json or module folder was set, then by default the main and bin entries in the package.json will be parsed, but you can add more the list of entries by passing them in as --entry. Supports globbing')
```

Comment:
Can we reverse this logic instead of overloading the functionality? E.g. We can support input of a list of paths and add a `--package` flag and/or a `--cwd` flag to override the old behaviors we think may still be useful. That way the default usage is just `dependency-check src/**/*.ts` or something,

Reply:
I think we can just drop `--entry` completely actually